modules/@apostrophecms/piece-type/ui/apos/components/AposImageCropper.vue

```diff
@@ -59,11 +82,12 @@ export default {
     }
   },
   created () {
-    this.setCoordinates = debounce(
-      (coordinates) => {
-        this.$refs.cropper.setCoordinates(coordinates);
-      }, 500
-    );
+    const DEBOUNCE_TIMEOUT = 500;
+
+    this.onScreenResizeDebounced = debounce(this.onScreenResize, DEBOUNCE_TIMEOUT);
```

Comment:
Some of these lines pretty long, could be indented

Reply:
I find it more readable than:

```js
this.onScreenResizeDebounced = debounce(
      this.onScreenResize,
      DEBOUNCE_TIMEOUT
    );
    this.handleCropperChangeDebounced = debounce(
      this.handleCropperChange,
      DEBOUNCE_TIMEOUT
    );
    this.setCropperCoordinatesDebounced = debounce(
      this.setCropperCoordinates,
      DEBOUNCE_TIMEOUT
    );
    this.updateFocalPointCoordinatesDebounced = debounce(
      this.updateFocalPointCoordinates,
      DEBOUNCE_TIMEOUT
    );
```

or

```js
    this.onScreenResizeDebounced =
      debounce(this.onScreenResize, DEBOUNCE_TIMEOUT);
    this.handleCropperChangeDebounced =
      debounce(this.handleCropperChange, DEBOUNCE_TIMEOUT);
    this.setCropperCoordinatesDebounced =
      debounce(this.setCropperCoordinates, DEBOUNCE_TIMEOUT);
    this.updateFocalPointCoordinatesDebounced =
      debounce(this.updateFocalPointCoordinates, DEBOUNCE_TIMEOUT);